.github/workflows/bin/spack-reviewers.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import re
+import subprocess
+
+import requests
+
+IS_PACKAGE = re.compile(r"repos/spack_repo/builtin/packages/([^/]+)/package.py$")
+
+
+def main():
+    repository = os.environ["GH_REPO"]
+    pr_number = os.environ["GH_PR_NUMBER"]
+    token = os.environ["GH_TOKEN"]
+
+    headers = {"Accept": "application/vnd.github+json"}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+
+    base_url = f"https://api.github.com/repos/{repository}/pulls/{pr_number}"
+    pull_request = requests.get(base_url, headers=headers).json()
+
+    pull_request_files = requests.get(f"{base_url}/files", headers=headers)
+
+    existing_reviewers = {reviewer["login"] for reviewer in pull_request["requested_reviewers"]}
+    maintainers = set()
+
+    for file in pull_request_files.json():
+        if match := IS_PACKAGE.match(file["filename"]):
+            package = match.group(1)
+
+            # add maintainers from packages here
+            result = subprocess.run(
+                ["spack", "maintainers", package], capture_output=True, text=True, check=False
+            )
+            maintainers.update(result.stdout.split())
+
+    author = pull_request["user"]["login"]
+    reviewers = (maintainers | existing_reviewers) - {author}
+
+    if existing_reviewers == reviewers:
+        print(f"[PR #{pr_number}]: labels already up-to-date")
```

Comment:
```suggestion
        print(f"[PR #{pr_number}]: reviewers already up-to-date")
```

Reply:
Fixed, I think right while you were doing your review haha